docker/Dockerfile

```diff
@@ -3,10 +3,6 @@ ARG PARENT_IMAGE=ubuntu:18.04
 # Garage base target ###########################################################
 FROM $PARENT_IMAGE AS garage-18.04
 
-# http://bugs.python.org/issue19846
-# > At the moment, setting "LANG=C" on a Linux system *fundamentally breaks Python 3*, and that's not OK.
-ENV LANG C.UTF-8
```

Comment:
hmm probably keep this until your dockerfile doesn't use Python 3.5.

Reply:
Oh right. I'll do this in the python 3.5 removal PR only then.